rustls/src/client/client_conn.rs

```diff
@@ -822,7 +824,9 @@ impl ConnectionCore<ClientConnectionData> {
         common_state.set_max_fragment_size(config.max_fragment_size)?;
         common_state.protocol = proto;
         common_state.enable_secret_extraction = config.enable_secret_extraction;
-        common_state.fips = config.fips();
+        // [FIPS REMOVED FROM THIS FORK]
+        // common_state.fips = config.fips();
```

Comment:
minor nit: slightly inconsistent with other updates - I would need to use block to uncomment this & use cfg condition to keep Rust stable happy, IIRC. I think it is best to keep it as it is now.